src/components/learner-credit-management/data/utils.js

```diff
@@ -218,10 +228,11 @@ export const formatPrice = (price, options = {}) => {
  */
 export const orderBudgets = (budgets) => {
   const statusOrder = {
-    Active: 0,
-    Scheduled: 1,
-    Expired: 2,
-    Retired: 3,
+    Expiring: 1,
+    Active: 1,
+    Scheduled: 2,
+    Expired: 3,
+    Retired: 4,
```

Comment:
[nit,curious] wondering if this is supposed to be in sequential order versus Expiring and Active both being labeled as 1?

Reply:
@adamstankiewicz , suggested that it shouldn't affect current order since both are basically the same thing with different labels. So setting them to both 1 preserve the current order.